docs/index.rst

```diff
@@ -6,11 +6,69 @@
 PlasmaPy Documentation
 ######################
 
+.. toctree::
+   :hidden:
+
+   /auto_examples/plot_distribution
+
+.. raw:: html
+
+    <div style="width: 30%;" class="sphx-glr-thumbcontainer" tooltip="Let&#x27;s try to look at ITER plasma conditions using the physics subpackage. ">
+
+.. only:: html
+
+    .. figure:: /auto_examples/images/thumb/sphx_glr_plot_physics_thumb.png
+
+        :ref:`sphx_glr_auto_examples_plot_physics.py`
+
+.. raw:: html
+
+    </div> 
+
+.. toctree::
+   :hidden:
+
+   /auto_examples/plot_physics
+
+.. raw:: html
+
+    <div style="width: 30%;" class="sphx-glr-thumbcontainer" tooltip="Let&#x27;s import some basics (and PlasmaPy!) ">
+
+.. only:: html
+
+    .. figure:: /auto_examples/images/thumb/sphx_glr_plot_dispersion_function_thumb.png
+
+        :ref:`sphx_glr_auto_examples_plot_dispersion_function.py`
+
+.. raw:: html
+
+    </div>
+
+.. toctree::
+   :hidden:
+
+   /auto_examples/plot_braginskii
+
+.. raw:: html
+
+    <div style="width: 30%;" class="sphx-glr-thumbcontainer" tooltip="Let&#x27;s analyze a few Langmuir probe characteristics using the diagnostics.langmuir subpackage. F...">
+
+.. only:: html
+
+    .. figure:: /auto_examples/images/thumb/sphx_glr_plot_langmuir_analysis_thumb.png
+
+        :ref:`sphx_glr_auto_examples_plot_langmuir_analysis.py`
+
+.. raw:: html
+
+    </div> 
+
```

Comment:
Ooooh.

Well, I feel dumb now for overlooking the "just add HTML" strategy, I was trying to get this done via sphinx gallery directives... This is perfectly fine and solves the issue of choosing examples though.